docs/Installation.md

```diff
@@ -24,24 +24,33 @@ This page describes how to use [AzerothCore with ArchLinux](arch-linux.md)
 
 Make sure your system fits the [Requirements](Requirements.md).
 
+**Note for Windows users**: use **git bash** as terminal to run commands (see Requirements).
```

Comment:
When installing on windows 10, I did not need to use git bash for anything other than the DB Assembler. There's already a note on the DB Assembler page about this.

Reply:
I'm not sure. I don't use Windows myself, but I created this PR after hearing the feedback from a user who couldn't figure it out

Reply:
Fair enough. Is there a public conversation I can look at so I can see exactly what issues they were running into? Perhaps they installed git incorrectly or something similar.

Reply:
Either way there shouldn't be any *harm* in using git bash so I don't think it'd be bad to add it.

Reply:
unfortunately it was not public